assets/css/button-outline.css

```diff
@@ -4,3 +4,7 @@
 	color: var(--wp--preset--color--base);
 	border-color: var(--wp--preset--color--contrast-2);
 }
+.wp-block-button.is-style-fill
```

Comment:
Can this be done in theme.json instead? button-outline.css will only load when that block style is being used on the page.